gitpod-io/gitpod
# [public-api-server] Forward Origin header where provided

## Description


## Related Issue(s)

Fixes #

## How to test
 - Go to https://gpl-papi-forward.preview.gitpod-dev.com/workspaces
 - open the network tab
 - create an Organization
 - notice all the public-api calls going through

## Release Notes

```release-note
NONE
```

## Documentation


## Build Options:

      Experimental feature to run the build with GitHub Actions (and not in Werft).
      leeway-target=components:all
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
